src/CognitoClient.php

```diff
@@ -494,15 +568,15 @@ public function verifyAccessToken($accessToken)
             throw new TokenVerificationException('invalid iss');
         }
 
-        if ($jwtPayload['token_use'] !== 'access') {
+        if ( !in_array($jwtPayload['token_use'], ['id','access']) ) {
             throw new TokenVerificationException('invalid token_use');
         }
 
         if ($jwtPayload['exp'] < time()) {
             throw new TokenExpiryException('invalid exp');
         }
 
-        return $jwtPayload['username'];
+        return $jwtPayload['username'] ?? $jwtPayload['cognito:username'];
```

Comment:
These changes are already in `master`, you need to update your branch.

Reply:
If you do an interactive rebase and remove the commits that aren't yours that should sort it.

Reply:
Did rebase for my branch